.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -46,17 +48,26 @@ jobs:
           echo "Date: $(date)" >> output.txt
           echo "Reference: $REF_URL" >> output.txt
           echo "Target version: $VERSION" >> output.txt
-          echo "-------------------------------------------" >> output.txt
+          echo "Chain: $CHAIN" >> output.txt
+          echo "----------------------------------------------------------------------" >> output.txt
+
+      - name: Pull polkadot-js-tools image
+        run: docker pull jacogr/polkadot-js-tools
```

Comment:
could add `--pull=always` to `docker run` below instead. But it doesn't make much sense since GHA are executed on the preemptible VMs, unless you'd bring docker cache with you.

Reply:
I actually tried using caching of the docker images with GHA which ended up a hassle for a very low gain in terms of time. So since I am not using caching `--pull=always` is implied. Do you see a benefit to add the option nonetheless? That would even get on the way to supporting caching later on if someone wants to give it a shot.

Reply:
> Do you see a benefit to add the option nonetheless?

Just to be more explicit.

> That would even get on the way to supporting caching

Then it's better to move it to GitLab CI anyway.

Reply:
done

Reply:
now, this step can be removed.

Reply:
reusing docker cache in GHA is hardly possible: it runs on preemptible VMs, hence the only way is to upload it in the previous job and download it here. Sum of these times sometimes is bigger than rebuilding the payload.

Reply:
I am considering reverting your suggestion actually as I would like to keep the 2 steps pull and run separate. It makes the logs more readable as the pull is quite verbose. So I aware we **could** have those 2 steps in one and I used to have it this way but it is better split.